### PR TITLE
[NIFI-13115] update context menu, search results overalay, and nf edi…

### PR DESCRIPTION
**Pull request by scottyaslan**
…tor overlay to match angular material overlay styles

### CODE REVIEW

**Comment by mcgilman** [comment]
Will review...

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss [hunk 1]
```diff
@@ -354,8 +353,9 @@
     }
 
     .property-editor {
-        background-color: $nifi-theme-surface-palette-darker;
-        box-shadow: 0 2px 5px rgba(0, 0, 0, 0.25);
+        background-color: var(--mat-menu-container-color);
+        box-shadow: 0 3px 6px rgba(0, 0, 0, 0.25);
+        border-radius: 4px;
```
Comment (mcgilman) [inline nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss:358]: The resize handle is rendered on top of the bottom right corner. We may want to consider updates there now that the modal uses a larger border radius.

Screenshot 2024-04-30 at 8 58 54 AM

**Review by mcgilman — COMMENTED** [review]
Thanks @scottyaslan! Just one concern I noted below.

File: nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss (hunk 1 above)
Comment (scottyaslan) [reply nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/assets/styles/_app.scss:357]: Good catch. I have updated it to be positioned like the other resized handles:
Screenshot 2024-04-30 at 12 37 55 PM

**Comment by scottyaslan** [comment]
@mcgilman Thanks for the review. I have rebased with latest main. Please merge once CI is green.

**Review by mcgilman — APPROVED** [review]
Thanks @scottyaslan!

**Pull request merged**